packages/assets-controllers/package.json

```diff
@@ -105,7 +105,7 @@
     "webextension-polyfill": "^0.12.0"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^20.0.0",
+    "@metamask/accounts-controller": "^21.0.0",
```

Comment:
It looks like we are bumping a peer dependency here. If we were to bump `accounts-controller` in a client that also uses `assets-controllers` then it would give us a peer dep warning since `assets-controllers` expects v20 but we now are using v21. 

Should we include this package in the release too (and bump it by a major)?

Reply:
The `assets-controllers` is already a major one yes, because of that peer dep. But indeed, I did miss the others (I thought the `create-release-branch` would automatically do it since I used a `major` bump for this controller, but nope 😅)

I'll do a manual changes for the dependent controllers

Reply:
Ah, gotcha, I missed `assets-controllers`.

> I thought the create-release-branch would automatically do it since I used a major bump for this controller, but nope 😅

I think this is a bug in `create-release-branch`. I'll file a ticket to investigate further.